### ISSUE TITLE
Chatbot: collect the name, not just the email address

### ISSUE THREAD

**Issue by georgejipa** [andrianvaleanu/WP-Maintenance-Mode#242]
Right now, the chatbot collects just the email address. But it should collect the name too.

Requests:
https://wordpress.org/support/topic/collect-name-with-bot/
https://wordpress.org/support/topic/chatbot/

**Comment by bibi-babi** [comment]
Is it not possible to add the name to the csv file as well? I do not think it is complicated to program since it is already inserted in a variable.

After all, it is quite normal to collect name (or nikcname) and email. Mailchlimp does too.

And then I don't understand the sense of asking to write a value (in this case the name but it could be anything) and then not use it ...
thank you

**Comment by georgejipa** [comment]
Yes, it is possible to add the name to the CSV file, but we have to make the bot collect it first. As I already stated, for now, only the email address is collected.

Anyway, the feature is already on our to-do list. No ETA, though.
